scripts/build.js

```diff
@@ -78,7 +80,11 @@ const buildSimpleIconsCssFile = (unicodeHexBySlug) => {
 }
 
 const buildSimpleIconsTtfFontFile = (svgFileContent) => {
-  const ttf = svg2ttf(svgFileContent, {});
+  const ttf = svg2ttf(svgFileContent, {
+    version: `Version ${packageJson.version.split('.').slice(0, 2).join('.')}`,
+    description: packageJson.description,
+    url: packageJson.homepage
+  });
```

Comment:
As per [the `svg2ttf` documentation](https://github.com/fontello/svg2ttf#api), we could also add a copyright string. Perhaps one based on the `packageJson.license` field. What do you think?

Reply:
The copyright is automatically discovered by svg2ttf from SVG `<metadata>CC0 1.0 Universal | simple-icons contributors</metadata>` tag.

Reply:
Good point, forgot that that was there 😅